examples/schema_explorer.py

```diff
@@ -46,14 +56,14 @@
 
 # since the graph is very big, we will generate an svg viz. of it
 gv_digraph.format = 'svg'
-gv_digraph.render(os.path.join(DATA_PATH, '', 'viz/HTAN-GV'), view=True)
+gv_digraph.render('data/gviz/HTAN-GV', view=True)
 print("The svg visualization of the entire schema has been rendered.")
 
 # graph visualization of a sub-schema
 seq_subgraph = schema_explorer.sub_schema_graph(TEST_CLASS, "up")
 
 seq_subgraph.format = 'svg'
-seq_subgraph.render('SUB-GV', view=True)
+seq_subgraph.render('data/gviz/SUB-GV', view=True)
```

Comment:
So I've summarized the minutes of a discussion we had in issue #328. We plan to breakdown the current contents of `examples/` into more atomic/independent units and add them to the `cli/` folder. So the first thing, is that this will be removed eventually, so there wasn't much use to me exposing this path through `config.yml`, and secondly, we decided only few parameters will be exposed through `config.yml` so as to not overwhelm the user.